include/zephyr/dt-bindings/regulator/cp9314.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) Cirrus Logic, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_REGULATOR_CP9314_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_REGULATOR_CP9314_H_
+
+/**
+ * @defgroup regulator_cp9314 CP9314 Devicetree helpers.
+ * @ingroup regulator_interface
+ * @{
+ */
+
+/**
+ * @name CP9314 Initial converter modes
+ * @{
+ */
+/** 2:1 mode */
+#define CP9314_MODE_2TO1 1
+/** 3:1 mode */
+#define CP9314_MODE_3TO1 2
+/** @} */
```

Comment:
are these used now? If for the new vendor property, please, add an example in the bindings file showing its usage.

Reply:
also, an enum on such property (if used there) to avoid user mistakes would be useful as well

Reply:
If you use enums & ENUM_IDX macros, you can remove this file.

Reply:
ack